src/common/layer_normalization_pd.hpp

```diff
@@ -215,7 +215,7 @@ struct layer_normalization_fwd_pd_t : public layer_normalization_pd_t {
                 + n_binary_po_inputs();
     }
     int n_outputs() const override {
-        return 1 + 2 * (!stats_are_src()) * is_training();
+        return (!stats_are_src() && is_training()) ? 3 : 1;
```

Comment:
I'd leave a comment that it works arounds a <specific version> of MSVC due to a bug.